module/zfs/spa.c

```diff
@@ -4197,20 +4363,22 @@ spa_import(char *pool, nvlist_t *config, nvlist_t *props, uint64_t flags)
 	return (0);
 }
 
-nvlist_t *
-spa_tryimport(nvlist_t *tryconfig)
+int
+spa_tryimport(nvlist_t *tryconfig, nvlist_t **newconfig)
 {
 	nvlist_t *config = NULL;
 	char *poolname;
 	spa_t *spa;
 	uint64_t state;
 	int error;
 
+	config = NULL;
```

Comment:
Config is already initialized to NULL, I believe you meant `*newconfig = NULL;` here.